app/src/main/res/values-de/strings-de.xml

```diff
@@ -485,8 +485,306 @@
+	<string name="send_data_to_developers">Sende die Daten zu den Entwickler über die Erfolgsquote verschiedener Geräte.</string>
+	<string name="send_display_glucose">Sende den angezeigten Glukosewert</string>
+	<string name="send_logs_to_developer">Sende Events- und Fehlerlogs automatisch an die Entwickler.</string>
+	<string name="send_snooze_to_all">Sende Snooze an alle</string>
+	<string name="sensor_age_or_expiry">Sensoralter oder -ablauf</string>
+	<string name="set_logging_appid">Setze die Logging-AppID</string>
+	<string name="set_sound_for_bg_alerts">Setze den Ton, der für Glukosealarme benutzt wird.</string>
+	<string name="share">Teilen</string>
+	<string name="short_speak_readings_shortcut">Zeige eine Verknüfpung im ☰ Menü zum Werte Sprechen.</string>
+	<string name="shortcut_to_bg_alerts">Verknüpfung zu den Glukosealarmen</string>
+	<string name="show_bolus_wizard_preview">Zeige die Bolus-Assistent-Vorschau</string>
+	<string name="show_calibration_accuracy">Zeige die Kalibrierungsgenauigkeit der letzten drei Tage.</string>
+	<string name="show_datatables_in_app_drawer">Zeige Kalibrierungs- und Glukosedatentabellen in der Navigationsleiste.</string>
+	<string name="show_expiry_time">Zeige die Sensorablauf Zeit basierend auf 14,5 Tagen.</string>
+	<string name="show_extra_status_on_widget">Zeige die Extrastatuszeile auch auf dem Widget.</string>
+	<string name="show_long_calibration_data">Zeige Steigung und Anfangswert in der langen Form.</string>
+	<string name="show_plugin_data">Zeige Steigung und Anfangswert vom aktiven Plugin.</string>
+	<string name="show_plugin_results_on_graph">Zeige Glukoseergebnisse vom Plugin auf dem Hauptgraphen.</string>
+	<string name="show_range_on_widget">Zeige eine Hoch- und eine Tieflinie auf dem Widget.</string>
+	<string name="show_reminder_features">Zeige Erinnerungsmenüs und -funktionen.</string>
+	<string name="show_sensor_age">Zeige das Alter des Sensors auf dem Hauptbildschirm</string>
+	<string name="show_short_calibration_data">Zeige die Steigung und den Anfangswert in kurzer Form.</string>
+	<string name="show_standard_deviation">Zeige Standardabweichung der Werte.</string>
+	<string name="show_test_features">Zeige Extra- und Alpha-Testfunktionen</string>
+	<string name="show_total_carbs">Zeige die Gesamtmenge an Behandlungskohlenhydraten.</string>
+	<string name="show_total_insulin">Zeige die Gesamtmenge an Behandlungsinsulin.</string>
+	<string name="skip_lan_uploads">LAN-Uploads überspringen</string>
+	<string name="skip_local_lan">Überspringe Uploads für lokale Server mit 192.168.x.x Adressen, wenn keine Verbindung zu einem lokalen Netzwerk besteht.</string>
+	<string name="snoozes_will_silence_all">Snoozes werden als Mobilteile in der Gruppe, die die Akzeptierungs-Einstellung unten gesetzt haben, verstummen lassen.</string>
+	<string name="speak_glucose_twice">Sprich den Glukosewert jedesmal zwei mal</string>
+	<string name="speak_readings_shortcut">Werte-Sprechen-Verknüpfung</string>
```

Comment:
Hier würde ich "Vorlesen" verwenden. Wie nennt Google das generell?
"Text in Sprache" gefällt mir aber auch nicht ;)